### PR TITLE
Edit basic concepts topic

### PR DESCRIPTION
**Pull request by guillermooo**

### CODE REVIEW

File: source/basic_concepts.rst [hunk 1]
```diff
@@ -2,53 +2,70 @@
...
-
-Sublime Text can't be mastered in a day, but it's built on a handful of
-pervasive ideas that make for a consistent and easily understandable system
+Unquestionably a versatile tool for programmers,
+you don't need to be one in order to
+use Sublime Text,
+or even to configure it extensively.
+If you're a hacker, however, you are in
+for a great many pleasant surprises:
+Sublime Text can be infinitely customized and extended.
```
Comment (FichteFoll) [inline source/basic_concepts.rst]: "infinitely" is an overstatement. Better use "highly" or "greatly".

File: source/basic_concepts.rst [hunk 2]
```diff
@@ -2,53 +2,70 @@
...
+You can start using it efficiently out of the box,
+but spending some time tailoring it
+to your exact needs will make it even better.
+
+This guide will teach you
+how to configure Sublime Text.
```
Comment (FichteFoll) [inline source/basic_concepts.rst]: And how it works.

File: source/basic_concepts.rst [hunk 3]
```diff
@@ -2,53 +2,70 @@
...
+
+Sublime Text can't be mastered in a day,
+but it's built on a handful of pervasive ideas
+that make for a consistent
+and easily understandable system
```
Comment (FichteFoll) [inline source/basic_concepts.rst:50]: I don't have a major in english, but I _feel_ like this should be hyphenated to "easily-understandable". Doesn't look that great either but I can't think of a better wording right now.

File: source/basic_concepts.rst [hunk 4]
```diff
@@ -57,127 +74,191 @@ This is a platform-dependent location:
...
+refers to the directory
+to which you've extracted the compressed portable files.
 
-Note that only in portable installations does a directory named *Data* exist.
-For the remaining installation types, the data directory is the location
+Note that only in portable installations
+does a directory named *Data* exist.
+For the remaining installation types,
+the data directory is the location
 indicated above.
```
Comment (FichteFoll) [inline source/basic_concepts.rst:83]: While I think this is poorly worded, we don't need this paragraph at all, imo.

File: source/basic_concepts.rst [hunk 5]
```diff
@@ -57,127 +74,191 @@ This is a platform-dependent location:
...
 The *Packages* Directory
 ==============================
 
-This is a **key directory**: all resources for supported programming and markup
-languages are stored here. A *package* is a directory or zip file containing
-related files having a special meaning for Sublime Text.
+This is a **key directory**:
+all resources for supported programming and markup languages
+are stored here.
+A *package* is a directory or zip file
```
Comment (FichteFoll) [inline source/basic_concepts.rst]: "archive"?

File: source/basic_concepts.rst [hunk 6]
```diff
@@ -57,127 +74,191 @@ This is a platform-dependent location:
...
+containing related files
+having a special meaning for Sublime Text.
 
-You can access the packages directory from the main menu
-(**Preferences | Browse Packages...**), or by means of an API call:
-``sublime.packages_path()``. In this guide, we refer to this location as
-*Packages*, *packages path*, *packages folder* or *packages directory*.
+You can access the packages directory
+from the main menu (**Preferences | Browse Packages...**),
```
Comment (FichteFoll) [inline source/basic_concepts.rst:98]: →

And we should use a true ellipses when it occurs in the menu label: …

File: source/basic_concepts.rst [hunk 7]
```diff
@@ -57,127 +74,191 @@ This is a platform-dependent location:
...
+while developing plugins.
+
+To open the Python console,
+press :kbd:`Ctrl+\``
+or select **View | Show Console**
 from the main menu.
 
-Confused? Let's try again more slowly:
+Confused?
+Let's try again more slowly:
```
Comment (FichteFoll) [inline source/basic_concepts.rst]: No.

File: source/basic_concepts.rst [hunk 8]
```diff
@@ -57,127 +74,191 @@ This is a platform-dependent location:
...
+and Python files too
+for the more advanced extensibility options.
+
+In this guide, for brevity,
+we refer collectively to all these
+disparate configuration files as *resources*.
+
+Sublime Text will look for resources
+inside the packages folder.
+And what is a package, you ask?
```
Comment (FichteFoll) [inline source/basic_concepts.rst]: I don't really like this kind of "interactive" guide style, but this one is not as bad as the others. Might also just be personal preference.

File: source/basic_concepts.rst [hunk 9]
```diff
@@ -57,127 +74,191 @@ This is a platform-dependent location:
...
-An evolution of Vintage called Vintageous_ offers a better Vi editing
-experience and is updated more often than Vintage. Vintageous_ is an open
-source project.
+An evolution of Vintage called Vintageous_
+offers a better Vi editing experience
+and is updated more often than Vintage.
+Vintageous_ is an open source project.
 
 .. _Vintage: http://www.sublimetext.com/docs/3/vintage.html
 .. _Vintageous: http://guillermooo.bitbucket.org/Vintageous
```
Comment (FichteFoll) [inline source/basic_concepts.rst:250]: Link to github repo instead?

**Comment by FichteFoll** [comment]
There's a bunch of other things I'd change here, but I'd do that in a separate pass. Prolly a commit.

**Comment by guillermooo** [comment]
I should have made more explicit that this is a WIP. I'm not still done with this one. Sorry about that!

File: source/basic_concepts.rst (hunk 9 above)
Comment (guillermooo) [inline source/basic_concepts.rst:250]: That's the home page, so it's fine. The repo is linked from there.

**Comment by guillermooo** [comment]
@FichteFoll PTAL

What do you think about using emphasis sparingly as in the file navigation topic?

**Comment by guillermooo** [comment]
BTW, I haven't yet, I'm just asking.

**Comment by FichteFoll** [comment]
> I should have made more explicit that this is a WIP. I'm not still done with this one. Sorry about that!

Just commenting on what I read~

**Comment by guillermooo** [comment]
> You changed the heading to "folder" but still use "directory" here. Why the change?

(Can't see your comment's in line for some reason.)

We use different names for it throughout the guide. No other reason that it seems to be more common these days as user-friendly term. Being this an introductory topic, it seemed a good idea.

**Comment by guillermooo** [comment]
> Can we call something portable an "installation" in the first place? Not sure about (official) terminology here but it does sound weird.

Good question. I'll try to reword.

**Comment by guillermooo** [comment]
> These, plus the paths in the list above, should use the :file: role.

Ok.

**Comment by guillermooo** [comment]
> Maybe state that ST itself will not overwrite (but other packages might)? (by which I mean we should add itself there)

Ok.

**Comment by guillermooo** [comment]
> Initially I thought of using :file:`{Packages}/User` for these cases.

I'll take a look at that.

**Comment by FichteFoll** [comment]
> (Can't see your comment's in line for some reason.)

That's because I'm reviewing and commenting on the actual commits and not the PR diff (which makes more sense here since we're iterating).

**Comment by guillermooo** [comment]
@FichteFoll I think I'm done with this topic. PTAL. If you think it's ready, I'll re-submit as a clean PR.

**Comment by FichteFoll** [comment]
Otherwise LGTM. Please re-consider the usage of bold text.

**Comment by guillermooo** [comment]
I'll resubmit and merge as a clean branch. Thanks!

We can do a second pass later.

**Pull request closed without merging**